sourcegit/api.py

```diff
@@ -43,6 +45,19 @@ def sync_upstream_pr_to_distgit(self, fedmsg_dict):
         with Synchronizer(self.pagure_user_token, self.pagure_package_token, self.pagure_fork_token) as sync:
             return sync.sync_using_fedmsg_dict(fedmsg_dict)
 
+    def keep_syncing_upstream_pulls(self):
+        """
+        Watch Fedora messages and keep syncing upstream PRs downstream. This runs forever.
+
+        :return: None
+        """
+        for topic, action, msg in self.consumerino.iterate_gh_pulls():
+            # TODO:
+            #   handle edited (what's that?)
```

Comment:
> what's that?

Maybe updates (=new commits) of the pull request?

Reply:
I think that "edited" means when someone updates title or content of the original post of a PR. But didn't have a chance to check it, hence the comment.

Reply:
Synchronize event is for new commits or a rebased PR.

Reply:
OK.